### ISSUE TITLE
Plot fatalities

### ISSUE THREAD

**Issue by GaelVaroquaux** [covid19-dash/covid-dashboard#67]
Several people (on twitter) asked a plot of the fatalities.

I think that this not should be overlayed to the exciting plot, as it is too dense. Rather, it would be interesting to add a button that switches the plot from active cases to fatalities.

**Comment by mrastgoo** [comment]
I would like to give this a try, it might take me sometimes though to figure it out.

**Comment by emmanuelle** [comment]
great Mojdeh, do you want to have a quick chat to talk about how to do this?

**Comment by mrastgoo** [comment]
yes, that would be great, I will be free after 6 pm tonight, btw I was thinking to use the last approach here https://plotly.com/python/custom-buttons/, basically creating a list of True and False for the visibility of countries within active, prediction and fatalities, respectively

**Comment by emmanuelle** [comment]
Yes, adding a custom button in order to toggle the visibility of active and fatalities traces would work!
